src/ec/google/chromeec/ec_commands.h

```
@@ -8939,6 +8939,51 @@ struct ec_response_get_boot_time {
 /* Issue AP shutdown */
 #define EC_CMD_AP_SHUTDOWN 0x0605
 
+/*****************************************************************************/
+/* Custom MrChromebox commands: range 0x3AC0-0x3ACF */
+
+/* Set and/or get After G3 State value. */
+#define EC_CMD_AFTER_G3_STATE 0x3AC0
+
+enum ec_after_g3_state {
```

why not have these align with the existing enum values in pmclib?

---

This is just my bad. At first I have reserved 0 for invalid/unknown value, and I have just stick to that later. It also aligned to a potential scenario where zeroing a variable would make it default to invalid/unknown value.

But you are right, aligning enum with pmclib is better.

Not only this allows to drop pmclib header, it also simplifies small arithmetic going on for name table array in EC code, where it is necessary to adjust value by 1 to match state enum (and vice-versa), so double-win here.

Thanks for pointing this out.

Fix:
https://github.com/MrChromebox/coreboot/pull/21/commits/66d1b6ae4289e902608fd4c91202dec4892d2172

Fix for EC:
https://github.com/MrChromebox/chrome-ec/pull/11/commits/49479777d068508709462cdcea37651a4c72df42

Is this ok?